tests/src/core/testqgsexpression.cpp

```diff
@@ -1640,6 +1640,25 @@ class TestQgsExpression: public QObject
       QTest::newRow( "array_last(array('a', 'b', 'c'))" ) << QStringLiteral( "array_last(array('a', 'b', 'c'))" ) << false << QVariant( "c" );
       QTest::newRow( "array_last(array())" ) << QStringLiteral( "array_last(array())" ) << false << QVariant();
 
+      // array_min, array_max, array_mean, array_median, array_majority, array_sum
+      QTest::newRow( "array_min(array())" ) << QStringLiteral( "array_min(array())" ) << false << QVariant();
```

Comment:
for safety, can you add tests where these functions are called with non-array arguments? Just to confirm that they don't result in crashes...

Reply:
I thought hey can't be called with non-array arguments without raising a syntax error

Reply:
Yeah that's what the test would confirm -- the "false" in these lines indicates whether a syntax error should be raised. (The logic for this sits within each individual expression function, so it's nice to include tests with unexpected types so that we avoid crashes)

Reply:
Added tests for int and strings. I messed git and had to force push though, sorry about that!

Reply:
perfect, thanks!